src/qt/locale/bitcoin_it.ts

```diff
@@ -89,7 +89,7 @@ E' possibile firmare solo con indirizzi di tipo "legacy".</translation>
     </message>
     <message>
         <source>Export Address List</source>
-        <translation>Esporta elenco degli indirizzi</translation>
+        <translation>Esporta la Lista degli Indirizzi</translation>
```

Comment:
Use downcase for those words: `lista` & `indirizzi`.
Should be like: `Esporta la lista degli indirizzi`

Reply:
```suggestion
        <translation>Esporta la lista degli indirizzi</translation>
```